src/api/Item.js

```diff
@@ -283,20 +289,22 @@ class Item extends Base {
      * API to create or remove a shared link
      *
      * @param {Object} item - Item to share
-     * @param {string} access - Shared access level
+     * @param {string|undefined} access - Shared access level; potentially undefined if there are no access changes
      * @param {Function} successCallback - Success callback
      * @param {Function|void} errorCallback - Error callback
      * @param {Array<string>|void} [options.fields] - Optionally include specific fields
      * @param {boolean|void} [options.forceFetch] - Optionally bypasse the cache
      * @param {boolean|void} [options.refreshCache] - Optionally update the cache
+     * @param {SharedLinkUpdateType} [sharedLinkRequestBody] - Optional custom request body for updating the shared link
```

Comment:
What is the purpose of this parameter? It doesn't appear to be used currently and seems to conflict with the `access` parameter. Should `share` be broken up into multiple methods to better support CRUD?

Reply:
Good catch. I meant to add this in my next PR. Will remove from this PR.